mods/cssb/moves.js

```diff
@@ -79,6 +81,7 @@ exports.BattleMovedex = {
 		},
 		target: "normal",
 		type: "Water",
+		desc: "For 5 turns, heavy rain powers Water moves and 20% chance to flinch the target."
```

Comment:
Shorten `For 5 turns, heavy rain powers Water moves and` to `Summons Rain and has`